### ISSUE TITLE
Required prop `children` was not specified && unknown props `onRequestAdd`, `onRequestDelete` on <input> tag

### ISSUE THREAD

**Issue by LittleCVR** [TeamWertarbyte/material-ui-chip-input#4]
I'm using material-ui-chip-input with React 15.3.1, Material-UI 0.15.4, the controlled example from the main page:

```
<ChipInput
  value={yourChips}
  onRequestAdd={(chip) => handleAddChip(chip)}
  onRequestDelete={(chip) => handleDeleteChip(chip)}
/>
```

gives me these warnings:

```
Warning: Failed prop type: Required prop `children` was not specified in `ChipInput`.
Warning: Unknown props `onRequestAdd`, `onRequestDelete` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
```

Probably the children prop shouldn't be marked as `isRequired` in PropTypes?

Also seems like `onRequestAdd` and `onRequestDelete` were passed by `...other` down to the input element.

**Comment by leMaik** [comment]
Thanks! I totally forgot updating the `propTypes` and (shame on me) ignored the warnings.

**Comment by leMaik** [comment]
Fixed in v0.3.1, will be published in a moment...

**Comment by LittleCVR** [comment]
Thanks for the quick fix! It complains no more about these 2 warnings, but I discovered a new one... will file another issue.
